src/node/entry.ts

```diff
@@ -126,19 +122,40 @@ const main = async (cliArgs: Args): Promise<void> => {
   }
 }
 
+function trimLDLibraryPath(): void {
+  let ldVar: string
+  if (process.platform === "linux") {
+    ldVar = "LD_LIBRARY_PATH"
+  } else if (process.platform === "darwin") {
+    ldVar = "DYLD_LIBRARY_PATH"
+  } else {
+    return
+  }
+
+  // Removes the leading path added by ./ci/build/code-server.sh to use our bundled
+  // dynamic libraries. See ci/build/build-standalone-release.sh
+  // This is required to avoid child processes using our bundled libraries.
+  process.env[ldVar] = process.env[ldVar]?.replace(path.dirname(process.execPath) + ":", "")
```

Comment:
I am not sure that `LD_LIBRARY_PATH` is populated at this point so it might be empty.  You can confirm correct operation if you run the following from a child process: 

```
ldd  $(which cmake) | grep code-server
```

if you see any output it is still pointing at code server then it might not have taken.  I think you might need to do something like AppImage where they rewrite the linker path for the binary to their library paths, that way you will not pollute downstream.  I am not sure if this https://medium.com/@nehckl0/creating-relocatable-linux-executables-by-setting-rpath-with-origin-45de573a2e98 is feasible.   That might be the best hope since you can put that in the installer when the location is selected.

Reply:
I built off this PR and can confirm there's no output for that command whereas there was before.